components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m

```diff
@@ -78,6 +84,17 @@ - (void)viewWillAppear:(BOOL)animated {
 - (void)viewDidLayoutSubviews {
   [super viewDidLayoutSubviews];
   _titleLabel.center = self.view.center;
+  UIEdgeInsets edgeInsets = UIEdgeInsetsZero;
+  if (@available(iOS 11.0, *)) {
+    edgeInsets = self.view.safeAreaInsets;
+  }
+  CGRect buttonFrame = self.buttonFrame;
+  self.button.frame = CGRectMake(buttonFrame.origin.x + edgeInsets.left,
```

Comment:
Please consider using `CGRectOffset`, since I believe that's what you're going for here.

Reply:
Addressed in a46864d6635c5903a535e6436aed7fb783f084b3.